source/docs/romi-robot/getting-to-know-romi.rst

```diff
@@ -102,6 +115,11 @@ PWM Output
 +-------------+-------------------------+
 | PWM 1       | Right Motor             |
 +-------------+-------------------------+
+| PWM 2       | Pin 21 / A3             |
```

Comment:
We should specify that this specific pin mapping is the default configuration, and that it can be configured via the web UI